src/mjolnir/convert_transit.cc

```diff
@@ -0,0 +1,1345 @@
+#include <cmath>
```

Comment:
nothing is changed here really this is just moved from the executable and the include paths updated. we might have changed the signature slightly to make it use the config only to do its work. the cool thing here is that now that its in the library we will be testing it and test coverage will inevitably go up. what i can reason about though is why it goes down in this pr. the executable was untested before so i assume it was also a coverage hit before..